docs/pages/includes/plugins/rbac-with-friendly-name.mdx

```diff
@@ -0,0 +1,65 @@
+Teleport's Access Request plugins authenticate to your Teleport cluster as a
+user with permissions to list and read Access Requests. This way, plugins can
+retrieve Access Requests from the Teleport Auth Service and present them to
+reviewers.
+
+Define a user and role called `access-plugin` by adding the following content to
+a file called `access-plugin.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: access-plugin
+spec:
+  allow:
+    rules:
+      - resources: ['access_request']
+        verbs: ['list', 'read']
+      - resources: ['access_plugin_data']
+        verbs: ['update']
+
+    # Optional: To display user-friendly names in resource-based access
+    # requests instead of resource IDs, the plugin also needs permission
+    # to list the resources being requested. Include this along with the
+    # list-access-request-resources role definition.
```

Comment:
```suggestion
    # Optional: To display user-friendly names in resource-based Access
    # Requests instead of resource IDs, the plugin also needs permission
    # to list the resources being requested. Include this along with the
    # list-access-request-resources role definition.
```